### DISCUSSION TITLE
snapclient input plugin ?

### DISCUSSION THREAD

**Discussion by pepie34** [MusicPlayerDaemon/MPD#1625] [Ideas]
Hi,

Exploring the code, in the Snapcast server output plugin there is a client class. Is there a snapclient input plugin not described in the documentation ?

If not available, it could be a good idea as the code seems there.

Best regards,

Pepie 34

**Comment by MaxKellermann** [comment]
No, what would it do?

**Reply by pepie34** [reply]
Just act as a snapclient, playing a stream coming from a snapserver from an other computer.
